src/views/CountryRiskWatch/index.tsx

```diff
@@ -1,8 +1,99 @@
+import { useParams, useOutletContext } from 'react-router-dom';
+
+import Container from '#components/Container';
+import Link from '#components/Link';
+import useTranslation from '#hooks/useTranslation';
+import useInputState from '#hooks/useInputState';
+import type { paths } from '#generated/riskTypes';
+import type { CountryOutletContext } from '#utils/country';
+import { useRequest } from '#utils/restRequest';
+
+import MultiMonthSelectInput from './MultiMonthSelectInput';
+import RiskTable from './RiskTable';
+import PossibleEarlyActionTable from './PossibleEarlyActionTable';
+import ReturnPeriodTable from './ReturnPeriodTable';
+import HistoricalDataChart from './HistoricalDataChart';
+
+import i18n from './i18n.json';
+import styles from './styles.module.css';
+import RiskBarChart from './RiskBarChart';
+
+type GetCountrySeasonal = paths['/api/v1/country-seasonal/']['get'];
+// FIXME: query type not available
+// type CountryRiskQuery = GetCountryRisk['parameters']['query'];
+type CountrySeasonal = GetCountrySeasonal['responses']['200']['content']['application/json'];
+
 // eslint-disable-next-line import/prefer-default-export
 export function Component() {
+    const { countryResponse } = useOutletContext<CountryOutletContext>();
+    const { countryId } = useParams<{ countryId: string }>();
+    const strings = useTranslation(i18n);
+    const [
+        selectedMonths,
+        setSelectedMonths,
+    ] = useInputState<Record<number, boolean> | undefined>({ 0: true });
+
+    const {
+        pending: pendingCountryRiskResponse,
+        response: countryRiskResponse,
+    } = useRequest<CountrySeasonal>({
+        apiType: 'risk',
+        url: '/api/v1/country-seasonal/',
+        query: {
+            iso3: countryResponse?.iso3?.toLowerCase(),
+        },
+    });
+
     return (
-        <div>
-            This is country Risk Watch
+        <div className={styles.countryRiskWatch}>
+            <Container
+                heading={strings.risksByMonthHeading}
+                headingLevel={2}
+                headerDescription={strings.risksByMonthDescription}
+                withHeaderBorder
+                childrenContainerClassName={styles.riskByMonthContent}
+            >
+                <MultiMonthSelectInput
+                    name={undefined}
+                    value={selectedMonths}
+                    onChange={setSelectedMonths}
+                />
+                <RiskTable
+                    riskData={countryRiskResponse?.[0]}
```

Comment:
Why are we only using the first data point? Let's add a NOTE

Reply:
It's also better to assign this to a variable as it's used multiple times.

Reply:
We only get 1 data point from the server, probably need to change the end-point

Reply:
Let's add a NOTE for now